README.md

```diff
@@ -155,19 +155,39 @@ modules in `components`, and `warn` level for the remaining codebase:
 RUST_LOG=casper_node::components::small=trace,casper_node::comp=info,warn
 ```
 
+### Logging network messages and tracing events
+
+Special logging targets exist in `net_in` and `net_out` which can be used to log every single network message leaving or
```

Comment:
Do we need to break this line? It will get wrapped correctly when displayed. These artifical break lines will mess up the presentation if you have wider screen.

Same comment applies for this whole file.

Reply:
> Same comment applies for this whole file.

I was faced with the same question and just went with what was there. I don't think this PR should address that. I also have no opinion on the matter :)